srcpkgs/applet-window-buttons/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'applet-window-buttons'
+pkgname=applet-window-buttons
+version=0.11.1
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -Wnodev .."
```

Comment:
these are already part of the build style (except for `-Wnodev` which just suppresses some warnings, so we should still leave it out)
```suggestion
```